docs/run/advanced-runtime-options.mdx

```diff
@@ -40,36 +46,103 @@ options.execution.shots = 2048
 ```
 ## Pass options to a primitive
 
-### Options class
+### Options class 
 
-When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you just created. Those options will then be applied when you use `run()` to perform the calculation.  Example: 
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example: 
 
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+    `SamplerV2` and `EstimatorV2` have separate options classes that do not need to be instantiated.  You can see the available options and update option values during or after primitive initialization.  Those options will then be applied when you use `run()` to perform the calculation.  Example: 
+
+```python
+estimator = Estimator(backend=backend)
+
+# Setting options after primitive initialization
+# This uses auto complete.
+estimator.options.optimization_level = 1
+estimator.options.default_shots = 4000
+```
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+When creating an instance of the `Estimator` or `Sampler` class, you can pass in the `options` you created in the options class. Those options will then be applied when you use `run()` to perform the calculation.  Example: 
 
 ```python
 estimator = Estimator(session=backend, options=options)
 result = estimator.run(circuit, observable).result()
 print(f">>> Metadata: {result.metadata[0]}")
 ```
+  </TabItem>
+</Tabs>
+
+### Primitive initialization (V2)
+
+With the V2 primitives, you can specify options when initializing the primitive. 
+
+ ```python 
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"resilience_level": 2})
+```
+
 ### Run() method
 
 You can pass in options by using the `run()` method. This overwrites the options you specified when creating the `Estimator` or `Sampler` instance for that particular execution. 
 
```

Comment:
In V1 you can pass _any_ options to `run()`. In V2 the only options you can pass to `run()` are options defined in the interface, i.e. `shots` for Sampler and `precision` for Estimator.